pkg/os/shell/shell_unix_test.go

```diff
@@ -78,7 +78,7 @@ func TestDetect_GivenPsOutputContainsShell_ThenReturnShellProcessWithMostRecentP
 
 			// Then
 			assert.Equal(t, "ps", mockCommandExecutor.commandName)
-			assert.Equal(t, []string{"-o", "pid=,comm=", "--sort=-pid"}, mockCommandExecutor.commandArgs)
+			assert.Equal(t, []string{"-o", "pid=,comm="}, mockCommandExecutor.commandArgs)
```

Comment:
This test `psCommandOutput` would need to be adjusted to have random order to verify that sorting works as expected.

I had changed the ordering to sorted as I expected we would receive the sorted list.

Reply:
Does it even matter because it is all mocked.

Reply:
Sorry, it's just my opinion as we're performing sorting on our own now. We can ignore it for now.